openadapt/start.py

```diff
@@ -0,0 +1,11 @@
+import subprocess
+
+result = subprocess.run(["git", "status"], capture_output=True, text=True)
+
+if "branch is up to date" not in result.stdout:
+    subprocess.run(["git", "stash"])
+    subprocess.run(["echo", "changes", "stashed"])
+    subprocess.run(["git", "pull"])
+    subprocess.run(["echo", "updated", "OpenAdapt"])
+
+subprocess.run(["python3", "-m", "app.main"], capture_output=True, text=True)
```

Comment:
would be better to directly call app.main (simpler / less overhead)

I have pushed a change to the add-gui PR so you should be able to do this:
```python
from openadapt.app.main import run_app

run_app() # start gui
```